packages/circuit-ui/components/Modal/components/ModalWrapper/ModalWrapper.tsx

```diff
@@ -19,7 +19,7 @@ import { css } from '@emotion/core';
 import styled, { StyleProps } from '../../../../styles/styled';
 import { Card, CardProps } from '../../../Card/Card';
 
-export type ModalWrapperProps = CardProps;
+export type ModalWrapperProps = Omit<CardProps, 'ref'>;
```

Comment:
Otherwise, TypeScript complains about conflicting types for `ref`. This component will be removed in v3 anyway.